components/Starknet/modules/tools/pages/limits_and_triggers.adoc

```diff
@@ -12,41 +12,35 @@ These are subject to revisions and change on a regular basis
-within each block to ensure block production times remain consistent and predictable.  | 8M | 8M
+within each block to ensure block production times remain consistent and predictable. | 5,000,000.0 | 5,000,000.0 | 5,000,000.0
 |Block limit (gas)| Certain Starknet operations, such as sending messages between L1 and L2, consume Ethereum gas. The current L1 state update
 mechanism involves an Ethereum transaction for each Starknet block.
 
 The gas limit for Starknet blocks is therefore inherited from the gas limit for Ethereum blocks.
-|3M |3M
+|5,000,000 |5,000,000 |5,000,000
 
 |Max transaction size (Cairo steps)|The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.
 This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.
-| 3M |3M
+| 3,000,000 | 3,000,000 | 3,000,000
 
-|Max Cairo steps for `validate`| The maximum number of computational steps, measured in Cairo steps, for a `validate` function.  |1M |1M
-
-|Gas cost for sending a message to L1| The cost in gas for sending a message from Starknet (L2) to Ethereum (L1) |5000 |5000
-
-|Gas cost for a storage read| The cost in gas for a storage read  |5000 |5000
-
-|Gas cost for a storage write| The cost in gas for a storage write|5000 |5000
+|Max Cairo steps for `validate`| The maximum number of computational steps, measured in Cairo steps, for a `validate` function. | 1,000,000 | 1,000,000 |1,000,000
 
 |Max contract bytecode size (No of felts in the program)| The maximum size of the bytecode or program that a smart contract can have on Starknet.
 
 Bytecode is the low-level code that comprises smart contracts. Limiting this size helps manage the complexity of contracts and the overall efficiency of the network.
-| 61440 | 61440
+| 61,440 | 61,440 | 61,440
 |Max contract class size|The maximum size for a contract class within Starknet.
 
 Contract classes are a fundamental building block for smart contracts, and limiting their size can have implications for the network's scalability and security.
-|  20MB | 20MB
+| 20MB | 20MB | 20MB
 |Mempool size limit |The mempool contains transactions that have undergone a preliminary verification by the sequencer
-and are candidates for inclusion into a block.|  No limitation|1000 transactions
+and are candidates for inclusion into a block. | No limitation | No limitation |1000 transactions
 |IP address limits (read)| In order to reduce network spam, Starknet limits the amount of contract reads that a single IP
-address can make. | 100 per min|200 per min
+address can make. | 100 per min| 100 per min|200 per min
 |IP address limits (write)|In order to reduce network spam, Starknet limits the amount of contract writes that a single IP
-address can make.| 10 per min / per ip address |10 per min / per ip address
+address can make.| 10 per min per IP address | 10 per min per IP address |10 per min per IP address
```

Comment:
since the entity name is `IP address limits (write)`, it already means `per ip address`, so imo, the following may be better:
```suggestion
address can make.| 10 per min | 10 per min |10 per min
```